pulsar-common/src/main/java/org/apache/pulsar/common/policies/data/LocalPolicies.java

```diff
@@ -52,4 +60,12 @@ public boolean equals(Object obj) {
         return false;
     }
 
+    @Override
+    public String toString() {
```

Comment:
Use `@ToString()` annotation on the class

Reply:
Also we could similarly take out the `equals()` and `hashCode()` methods.

Reply:
thanks for reply. i'll change this.